apps/formbricks-com/pages/docs/integrations/zapier/index.mdx

```diff
@@ -0,0 +1,110 @@
+import { Layout } from "@/components/docs/Layout";
+import { Fence } from "@/components/shared/Fence";
+import { Callout } from "@/components/shared/Callout";
+import Image from "next/image";
+
+import AddApiKey from "./add-api-key.png";
+import AddNewZap from "./add-new-zap.png";
+import ApiKeySecret from "./api-key-secret.png";
+import ChooseEvent from "./choose-event.png";
+import ConnectWithFB1 from "./connect-with-formbricks-1.png";
+import ConnectWithFB2 from "./connect-with-formbricks-2.png";
+import DuplicateSurvey from "./duplicate-survey.png";
+import SelectSurvey from "./select-survey.png";
+import SlackChannelMsg from "./slack-channel-msg.png";
+import SlackMsg from "./slack-message.png";
+import SubmitTestResponse from "./submit-test-response.png";
+import SuccessConnection from "./success-connected.png";
+import TestSubmission from "./test-submission.png";
+import UpdateQuestionId from "./update-question-id.png";
+import ZapierMessage from "./zapier-message.png";
+
+export const meta = {
+  title: "Zapier Setup",
+  description: "Wire up Formbricks with Zapier and 5000+ other apps",
+};
+
+Zapier is a powerful ally. Hook up Formbricks with Zapier and you can send your data to 5000+ other apps. Here is how to do it.
+
+<Callout title="Nail down your survey first" type="note">
+  Any changes in the survey cause additional work in the Zap. It makes sense to first settle on the survey you
+  want to run and then get to setting up Zapier.
+</Callout>
+
+## Step 1: Setup your survey incl. `questionId` for every question
+
+When setting up the Zap your life will be easier when you change the `questionId`s of your survey questions. You can only do so **before** you publish your survey.
+
+<Image src={UpdateQuestionId} alt="Update Question ID" quality="100" className="rounded-lg" />
+
+_In every question card in the Advanced Settings you find the Question ID field. Update it so that you’ll recognize the response tied to this question._
+
+<Callout title="Already published? Duplicate survey" type="note">
+  You can only update the questionId when the survey was not yet published. Already published it? Just
+  **duplicate it** to update the questionIds.
+  <Image src={DuplicateSurvey} alt="Duplicate Survey" quality="100" className="rounded-lg" />
+</Callout>
+
+## Step 3: Send a test response
+
+In order to set up Zapier you’ll need a test response. This allows you to select the individual values of each response in your Zap. If you want to set up an in-app survey, you can use our [Demo App](/docs/contributing/demo) to trigger a survey and submit a response that without much setting. It also works with Formbricks Cloud.
```

Comment:
I wouldn't mention the Demo app here as it is no option for most of the users (who haven't setup the repository and/or have no knowledge how to do so).

Reply:
not sure if it helps to not inlude any info about how to get a test submission for an in-app survey. I have reworded it a bit :)